source/gui/resources/daquiri.qrc

```diff
@@ -95,5 +95,7 @@
         <file>blank16.png</file>
         <file>fonts/Ruben.ttf</file>
         <file>fonts/SF_Fedora_Titles.ttf</file>
+        <file>kafka.png</file>
+        <file>kafka16.png</file>
```

Comment:
I'm not sure we can use these images. Apache made other software projects stop using the name Kafka in their name (e.g. Kafka Manager became CMAK).

Reply:
Well, since the project name has nothing with Kafka in it and the logo does not appear very prominently. I'd be inclined to let it pass for now. After all there is a myriad of projects with Kafka in their name on guthub. We even have kafkacow. We can always change this on request.